.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## PR Checklist
+Please check that your PR fulfills the following requirements:
+
+- [ ] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
```

Comment:
LGTM just one change, should have 1 more checkbox for linting passes.

Reply:
Also for gifs/videos for frontend PRs

Reply:
> LGTM just one change, should have 1 more checkbox for linting passes.

That will not be needed, as circle CI will have the yarn lint tests once eslint PR is pushed.